templates/web/prod/actions/Tools/videoEditor.html.twig

```diff
@@ -56,12 +59,30 @@
                                     {% set ratio = '' %}
                                 {% endif %}
 
-                                <video id="thumb_video" controls="" preload="auto" data-ratio="{{ ratio }}">
+                                {#Set the document Ratio #}
+                                {% for subdef in previewHtml5 %}
+
+                                    {% set width = subdef.get_width()  %}
+                                    {% set height = subdef.get_height() %}
+
+                                    {% if width and height %}
+                                        {% set rawRatio =  (width / height)|number_format(3, '.') %}
+                                        {% set rawRatioLength =  rawRatio|length %}
+                                        {% if subdef.get_name()=='preview' %}
+                                            {% set prevRatio = rawRatio|slice(0,rawRatioLength-1) %}
+                                        {% endif %}
+                                    {% else %}
+                                        {% set prevRatio = '' %}
```

Comment:
adding data-ratio="" may have side effects (?), it's better not to add the attribute at all.

Reply:
data-ratio is needed on javascript part to generate the canvas size